### PR TITLE
New Tutorial: Synchronous Request-Response with Amazon API Gateway and AWS Lambda

### PR DESCRIPTION
**Pull request by DennisTraub**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by DennisTraub** [comment]
Some images can't be found in the preview. I don't know why that is the case. I've double-checked and they are all part of the PR.

**Comment by caller79** [comment]
Some image paths are incorrect, such as (images\screen-new-stack.png).

URL's use / as path separator, not \

**Comment by DennisTraub** [comment]
Oh, man Windows! Thanks @caller79, I'll fix...

**Comment by DennisTraub** [comment]
Done. Learning: ➡ Don't drag and drop images into your file when using Windows

**Comment by scribwell** [comment]
@bketelsen This article is ready for your technical review.

**Review by bketelsen — CHANGES_REQUESTED** [review]

File: tutorials/application-integration-patterns/01-request-response-with-api-gateway/index.md [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+
+## Introduction
+
+The request-response pattern is a widely used communication model in modern web and cloud architectures. It facilitates synchronous interactions between clients and servers, providing real-time responses and immediate feedback when needed.
+
+This pattern is straightforward, easy to understand, and serves as the foundation for the largest distributed system in the world: the Internet itself. Whenever you open a web page or click on a link in your browser, your browser sends a request to the server, which in turn responds with an HTML document.
+
+In this step-by-step tutorial, we will implement this pattern using Amazon API Gateway and an AWS Lambda function. 
+
+![](images/diagram-api-gw-lambda.png)
```
[the remark this answers is not in the document]
Comment (DennisTraub) [reply tutorials/application-integration-patterns/01-request-response-with-api-gateway/index.md]: Oops, I must have missed that one. Added 👍

**Review by bketelsen — APPROVED** [review]

**Pull request merged**